go/pkg/client/bytestream.go

```diff
@@ -15,19 +15,21 @@ import (
 )
 
 // WriteBytes uploads a byte slice.
-func (c *Client) WriteBytes(ctx context.Context, name string, data []byte) error {
+func (c *Client) WriteBytes(ctx context.Context, name string, data []byte) (int64, error) {
```

Comment:
Actually, do we need to change this one? I don't think we use it in UloadIfMissing. Let's only change the things we have to change for UploadIfMissing.

Reply:
I don't think we need to change this one, but this sounds like delaying until we inevitably want more metrics?
Done